docs/local-development.md

```diff
@@ -238,21 +238,21 @@ The Debugger supports two types of internationalization RTL (right to left) layo
 
 #### L10N
 
-[L10N](https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/src/utils/L10N.js) is a global module with two methods `getStr` and `getFormatStr`.
+[L10N][l10n] is a global module with two methods `getStr` and `getFormatStr`.
```

Comment:
Markdown pro-tip! You can just define `[L10N]` at the bottom instead of creating the lowercase `[l10n]` alias and then here simply use `[L10N]` without anything else.